src/main/groovy/com/avast/gradle/dockercompose/ComposeExtension.groovy

```diff
@@ -125,6 +130,10 @@ class ComposeExtension {
     boolean removeOrphans() {
         getDockerComposeVersion() >= VersionNumber.parse('1.7.0') && this.removeOrphans
     }
+    
+    boolean scale() {
+        getDockerComposeVersion() >= VersionNumber.parse('1.13.0') && this.scale

```

Comment:
We should throw an exception if somebody set `scale` but they don't have Docker Compose >= 1.13.0.